src/sql/util.py

```diff
@@ -157,6 +157,84 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates
+) -> bool:
+    """
+    Raises UsageError when duplicate arguments are passed to magics.
+    Returns true if no duplicates in arguments or aliases.
+    """
+
+    aliased_arguments = {}
+    unaliased_arguments = []
+
+    for decorator in magic_execute.decorators:
+        assert isinstance(decorator.args, tuple)
```

Comment:
`assert` will raise unhandled exception in case it's false. this can be done in if else

Reply:
Removed assert as it was not needed [here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R195-R201)